src/styl/_helpers/colors.twig

```diff
@@ -1,3 +1,80 @@
-{% for key, name in colors %}
-    <p class="color-{{ key }}">Exemple de texte couleur {{ name }}</p>
-{% endfor %}
+<style>
+    .co_colors {
+        margin-bottom: 2rem;
+    }
+    .co_colors-item {
+        font-size: 14px;
+        padding: .3rem;
+        margin: .5rem;
+        text-align: center;
+    }
+    .co_color-demo {
+        height: 3em;
+        border-radius: 0.5em;
+    }
+    .co_color-caption {
+        padding: 0.2em;
+    }
+    .co_color-caption strong{
+        display: block;
+    }
+    .border-item {
+        border-width: 2px;
+        border-style: solid;
+    }
+</style>
+
+<strong>Text color</strong>
+<ul class="co_colors">
```

Comment:
we should not use `co_` classes in colette it should be reserved to colette-kss-builder.